modules/programs/ssh.nix

```diff
@@ -125,6 +125,14 @@ let
         description = "The command to use to connect to the server.";
       };
 
+      addressFamily = mkOption {
+        default = null;
+        type = types.enum [null "any" "inet" "inet6"];
```

Comment:
don't list null as part of the enum, instead use `types.nullOr types.enum [null "any" "inet" "inet6"];`

Reply:
It gives error on 18.09:

error: attempt to call something which is not a function but a set, at /home/[redacted]/.config/nixpkgs/home-manager/modules/programs/ssh.nix:138:16                                                                     
(use '--show-trace' to show detailed location information)

Reply:
@dsx You need a parenthesis: `types.nullOr (types.enum ["any" "inet" "inet6"]);`

Reply:
@dsx my apologies for the confusion, I forgot about currying :)